app/lib/rocketchat.js

```diff
@@ -1166,6 +1166,10 @@ const RocketChat = {
 		return !isUnread;
 	},
 
+	isSwipeEnabled(item) {
```

Comment:
Imo we shouldn't use `rocketchat.js` for something related to a component only.
Thinking on colocation, this function should be on RoomsListView, just like `isGrouping`.